phalcon/Http/Request.zep

```diff
@@ -845,16 +845,31 @@ class Request implements RequestInterface, InjectionAwareInterface
     }
 
     /**
-     * Gets HTTP URI which request has been made
+     * Gets HTTP URI which request has been made to
+     *
+     *```php
+     * // Returns /some/path?with=queryParams
+     * $uri = $request->getURI();
+     *
+     * // Returns /some/path
+     * $uri = $request->getURI(true);
+     *```
+     *
+     * @param bool onlyPath If true, query part will be omitted
+	 * @return string
```

Comment:
Looks like a combination of tabs and spaces mixing up that last part of the docblock.